lib/functions

```diff
@@ -210,3 +210,22 @@ function configure_service {
     chmod +x /etc/init.d/$service_script
     update-rc.d $service_script defaults
 }
+
+# Install OpenDayLight SDN controller - Download and install ODL
+function install_ODL {
+    if [ ! -d /opt/opendaylight ]; then
+        mkdir /opt/opendaylight/
+        wget https://nexus.opendaylight.org/content/repositories/public/org/opendaylight/integration/distribution-karaf/0.5.3-Boron-SR3/distribution-karaf-0.5.3-Boron-SR3.tar.gz -P /opt/
+        tar xvf /opt/distribution-karaf-0.5.3-Boron-SR3.tar.gz -C /opt/
+        mv /opt/distribution-karaf-0.5.3-Boron-SR3 /opt/opendaylight/current
+        rm -rf distribution-karaf-0.5.3-Boron-SR3.tar.gz
+    fi
+}
+
+# Start OpenDayLight SDN controleer - Start OpenDayLight
+function start_ODL {
+    if [ -d /opt/opendaylight ]; then
+        /opt/opendaylight/current/bin/start
```

Comment:
Usually the karaf server takes time to be up and running, I suggest to add a delay to wait for it before installing features.